tools/interop_matrix/create_testcases.sh

```diff
@@ -52,11 +55,18 @@ function cleanup {
   [ -e "$CMDS_SH" ] && rm $CMDS_SH
 }
 trap cleanup EXIT
+# TODO(adelez): add test auth tests but do not run if not testing on GCE.
 # Running the testcases as sanity unless we are asked to skip.
 [ -z "$SKIP_TEST" ] && (echo "Running test cases: $CMDS_SH"; sh $CMDS_SH)
 
+# Convert c++ to cxx. 
+client_lang=$LANG
+if [ $LANG == "c++" ]
```

Comment:
nit (if-then style consistency with the rest of the code and avoid repeated assignment) :
  
if [$LANG == "c++" ]; then
  client_lang="cxx"
else
  client_lang=$LANG
fi

Reply:
Done.